sysutils/scp-backup/src/opnsense/mvc/app/models/OPNsense/ScpBackup/General.xml

```diff
@@ -0,0 +1,40 @@
+<model>
+    <mount>//OPNsense/ScpBackup</mount>
+    <description>
+        scp-backup - a simple plugin to backup the config using SCP.
+    </description>
+    <version>1.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <hostname type="TextField">
+          <default></default>
+          <Required>Y</Required>
+          <ValidationMessage>Please provide a hostname</ValidationMessage>
+        </hostname>
+        <port type="IntegerField">
+          <default>22</default>
+          <Required>N</Required>
+          <MinimumValue>1</MinimumValue>
+          <MaximumValue>65535</MaximumValue>
+          <ValidationMessage>Please provide a valid port number between 1 and 65535. Port 22 is usually the default.</ValidationMessage>
+        </port>
+        <username type="TextField">
+          <Required>N</Required>
+          <ValidationMessage>Please provide a username.</ValidationMessage>
+        </username>
+        <remoteLocation type="TextField">
+          <Required>N</Required>
+          <ValidationMessage>Please provide a remote location.</ValidationMessage>
```

Comment:
directory

Reply:
Happy to change it to RemoteDirectory :-)

Reply:
please no camelcase in strings - that's already hard to read in the code ;)